core/src/main/scala/cromwell/core/callcaching/docker/DockerClientHelper.scala

```diff
@@ -0,0 +1,27 @@
+package cromwell.core.callcaching.docker
+
+import akka.actor.{Actor, ActorLogging, ActorRef}
+import cromwell.core.actor.RobustClientHelper
+import cromwell.core.callcaching.docker.DockerHashActor.DockerHashResponse
+
+import scala.concurrent.duration.FiniteDuration
+
+trait DockerClientHelper extends RobustClientHelper { this: Actor with ActorLogging =>
+  
+  protected def dockerHashingActor: ActorRef
+  
+  private [core] def dockerResponseReceive: Receive = {
```

Comment:
I don't know who started the idea, but I've heard that in the future, most people will fall into two camps: either telling the computers what to do, or the computers telling you what to do. In this case, I bow.